gadgets/chaos_packet_drop/program.bpf.c

```diff
@@ -0,0 +1,385 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2022 Microsoft Corporation
+
+#include <vmlinux.h>
+
+/* I am directly including the 
+value of the constants instead of 
+the linux/if_ether.h header file and linux/pkt_cls.h
+because of redeclaration conflicts with
+vmlinux.h */
+
+#define ETH_P_IP 0x0800 /* Internet Protocol packet	*/
+#define ETH_P_IPV6 0x86DD /* IPv6 over bluebook		*/
+
+#define TC_ACT_UNSPEC (-1)
+#define TC_ACT_OK 0
+#define TC_ACT_RECLASSIFY 1
+#define TC_ACT_SHOT 2
+#define TC_ACT_PIPE 3
+#define TC_ACT_STOLEN 4
+#define TC_ACT_QUEUED 5
+#define TC_ACT_REPEAT 6
+#define TC_ACT_REDIRECT 7
+
+#include <stdbool.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_endian.h>
+#include <gadget/types.h>
+#include <gadget/macros.h>
+#include <gadget/buffer.h>
+
+#define GADGET_TYPE_NETWORKING
+#include <gadget/sockets-map.h>
+
+/* The maximum number of items in maps */
+#define MAX_ENTRIES 8192
+#define TASK_COMM_LEN 16
+
+struct events_map_key {
+	struct gadget_l4endpoint_t dst;
+};
+
+struct event {
+	gadget_timestamp timestamp_raw;
+	struct gadget_l4endpoint_t src;
+	gadget_counter__u32 drop_cnt;
+	bool ingress;
+	bool egress;
+
+	gadget_netns_id netns_id;
+	struct gadget_process proc;
+};
+
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
+	__uint(max_entries, MAX_ENTRIES);
+	__type(key,
+	       struct events_map_key); // The key is going to be L4 gadget endpoint
+	__type(value, struct event);
+} events_map SEC(".maps");
+
+GADGET_MAPITER(events_map_iter, events_map);
+
+// we use the following variables as parameters
+const volatile struct gadget_l3endpoint_t filter_ip = { 0 };
+const volatile __u16 port = 0;
+const volatile __u32 loss_percentage = 100;
+const volatile bool filter_tcp =
+	true; /* This is a boolean flag to enable filtering of TCP packets */
```

Comment:
nit: What about something like 

```
/* This is a boolean flag to enable filtering of UDP packets */
const volatile bool filter_udp = true;
```

to avoid this weird formatted caused by the linter?